kbase-extension/static/kbase/css/errorPage.css

```diff
@@ -0,0 +1,37 @@
+#header,
+#site {
+  display: block;
+}
+
+.access-error__container,
+.generic-error__container {
+  text-align: center;
+  margin: 2rem;
+}
+
+.access-error__heading,
+.generic-error__heading {
+  font-weight: bold;
+  font-size: 200%;
+  line-height: 1.5;
```

Comment:
This is all a bit random and horrible, but leaving it as-is for now. When the typography is sorted out, it can be edited to be more uniform.